tool/bindings_generator/filesystem_api.dart

```diff
@@ -33,5 +43,19 @@ extension FileSystemExtension on FileSystem {
     }
   }
 
+  @JS('readFileSync')
```

Comment:
I believe you don't need both, you can make the second argument optional.

Calls to external JS interop functions only pass in the arguments in the call-site, so you still need the two different calls to `readFileSync` if you don't want to pass `null` for `options`, but you don't need both external members.

Reply:
I switch this to just:

```
external JSAny readFileSync(JSString path, [JSReadFileOptions options]);
```

and it seems to work fine. Thanks! That's much simpler.